frontend/src/App/AppUpdatedModalContent.js

```diff
@@ -76,7 +76,7 @@ function AppUpdatedModalContent(props) {
             <div>
               {
                 !update.changes &&
-                  <div className={styles.maintenance}>Maintenance release</div>
+                  <div className={styles.maintenance}>Maintenance Release: See Github Commit History for the technical details.</div>
```

Comment:
```suggestion
                  <div className={styles.maintenance}>Maintenance Release: See GitHub commit history for details.</div>
```

This corrects capitalization and in retrospect we don't need technical details. A link directly to the commits could be useful though.